### PR TITLE
Fix the missing deps issue from the bash script.

### LINKED ISSUE #40: ./nord.sh: line 275: missing_deps: unbound variable
When running this shell script, get this error message when running.

### PR DESCRIPTION
**Pull request by dsmithhayes**
Fixes #40

### CODE REVIEW

File: src/nord.sh [hunk 1]
```diff
@@ -266,13 +266,14 @@ print_help() {
 # @return 0 if all required dependencies are validated, 1 otherwise
 # @since 0.2.0
 validate_dependencies() {
-  declare -a missing_deps deps=("${!1}")
+  declare -a deps=("${!1}")
+  declare -a missing_deps=()
```
Comment (svengreb) [inline src/nord.sh]: Let's assign the value separately which is safer for Bash.

```suggestion
  declare -a missing_deps deps
  deps=("${!1}")
  missing_deps=()
```

File: src/nord.sh [hunk 2]
```diff
@@ -266,13 +266,14 @@ print_help() {
...
   for exec in "${deps[@]}"; do
     if ! command -v "${exec}" > /dev/null 2>&1; then
       missing_deps+=(${exec})
     fi
   done
-  if [ ${#missing_deps[*]} -eq 0 ]; then
+  if [ "${#missing_deps[*]}" -eq 0 ]; then
```
Comment (svengreb) [inline src/nord.sh]: This can lead to problems for the iterator values so let's revert this.

```suggestion
  if [ ${#missing_deps[*]} -eq 0 ]; then
```

**Review by svengreb — CHANGES_REQUESTED** [review]
Hi @dsmithhayes :wave:, thanks for your contribution :+1:
I've added some small suggestions, but otherwise this is the fix I've also mentioned in #40 💪

**Comment by dsmithhayes** [comment]
Hello @svengreb, I apologize for the late response, but I have made the requested changes.

**Review by svengreb — APPROVED** [review]
@dsmithhayes There's no need for an apology, you don't owe anyone anything in open source since anyone spend free time 😉
Thanks for updating the changes, this looks good to me 🚀

**Pull request merged**